src/components/layout/Header.js

```diff
@@ -0,0 +1,48 @@
+import { Title } from "../common/Title_Styled";
+import { SearchBar } from "../common/SearchBar";
+import { Icon } from "../common/Icon";
+import styled, { css } from "styled-components";
+
+const StyledHeader = styled.header`
+  align-items: center;
+  padding: 15px 20px;
+
+  .row {
+    align-items: center;
+  }
+
+  .shoppig-cart-container a {
+    color: black;
+  }
+
+  .shoppig-cart-container svg {
+    width: 30px;
+    height: 30px
+  }
+`;
```

Comment:
A common thing in react is to have a folder for each component, for example the `Header` could live in a folder called `Header` and inside the folder you can have the following structure:

- Header/
   - Header.js
   - Header.test.js
   - Header.styled.js
   - index.js

The `Header.js` can contain the component definition, `Header.test.js` is meant for the test cases of the component, and the `Header.styled.js` contains the styled components. The `index.js` just has the export of the component so when using it you can do it like `import Header from '../Header' instead of having to do `import Header from '../Header/Header.js'